### PR TITLE
New e2e tests and fix

### PR DESCRIPTION
**Pull request by Serhy**
- two tests added
- MultipleDeviceTestCase change is so that self.errors are reachable not only after code compile

### CODE REVIEW

File: test/appium/tests/base_test_case.py [hunk 1]
```diff
@@ -247,11 +246,12 @@ def teardown_class(cls):
         cls.loop.close()
 
 
-environment = LocalMultipleDeviceTestCase if pytest.config.getoption('env') == 'local' else SauceMultipleDeviceTestCase
-
-
-class MultipleDeviceTestCase(environment):
-    pass
+if pytest.config.getoption('env') == 'local':
+    class MultipleDeviceTestCase(LocalMultipleDeviceTestCase):
```
Comment (antdanchenko) [inline test/appium/tests/base_test_case.py]: you can do `MultipleDeviceTestCase = LocalMultipleDeviceTestCase`

File: test/appium/tests/base_test_case.py [hunk 2]
```diff
@@ -247,11 +246,12 @@ def teardown_class(cls):
...
+        pass
+else:
+    class MultipleDeviceTestCase(SauceMultipleDeviceTestCase):
```
Comment (antdanchenko) [inline test/appium/tests/base_test_case.py]: MultipleDeviceTestCase = SauceMultipleDeviceTestCase

File: test/appium/tests/atomic/transactions/test_wallet.py [hunk 3]
```diff
@@ -426,6 +426,28 @@ def test_set_currency(self):
...
+        send_transaction.element_by_text("Status account").click()
+        send_transaction.amount_edit_box.click()
+        send_transaction.amount_edit_box.set_value("0")
+        send_transaction.confirm()
+        send_transaction.sign_transaction_button.click()
+        send_transaction.cancel_button.click()
+        send_transaction.toggle_airplane_mode()
+        send_transaction.sign_transaction_button.click()
+        if send_transaction.sign_with_password.is_element_displayed():
+            pytest.fail("Sign transaction button is active in offline mode")
```
Comment (antdanchenko) [inline test/appium/tests/atomic/transactions/test_wallet.py]: please use self.driver.fail if you want proper logging

File: test/appium/tests/atomic/dapps_and_browsing/test_browsing.py [hunk 4]
```diff
@@ -123,3 +123,14 @@ def test_refresh_button_browsing_app_webview(self):
...
+    @marks.testrail_id(5785)
+    @marks.critical
+    def test_can_open_dapp_from_dapp_store(self):
+        sign_in = SignInView(self.driver)
+        home = sign_in.create_user()
+        daap_view = home.dapp_tab_button.click()
+        dapp_store_view = daap_view.discover_dapps_button.click()
+        dapp_store_view.element_by_text_part("CryptoKitties").click()
+        if not dapp_store_view.element_by_text_part("Start", "text").is_element_displayed(20):
+            self.pytest.fail("Failed to access CryptoKitties Dapp from Discover Dapp store")
```
Comment (churik) [inline test/appium/tests/atomic/dapps_and_browsing/test_browsing.py]: self.driver.fail here?

File: test/appium/tests/base_test_case.py (hunk 1 above)
Comment (Serhy) [reply test/appium/tests/base_test_case.py]: Agree, makes sense here, thanks @antdanchenko

**Review by churik — APPROVED** [review]

**Pull request merged**